exercises/isogram/cases_test.go

```diff
@@ -69,4 +74,9 @@ var testCases = []struct {
 		input:       "angola",
 		expected:    false,
 	},
+	{
+		description: "a non letter before a repeating pattern",
+		input:       "red bat cat",
+		expected:    false,
+	},
```

Comment:
This is a good additional test case.  I see it has not made it into [master of isogram/canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/canonical-data.json) but is in your [branch](https://github.com/da-edra/problem-specifications/commit/f49969844fe8da2ba5a8a7b24d912ae32ed2ab61#diff-4a2164ce55eed791d3cb1cb39985696a). Perhaps re-open the PR for it there? You may need a less "Go specific" description/log message.

Reply:
Thank you very much Lee, I decided to delete this test as the `thumbscrew-jappingly` case already covers the  situation were a `break` and a `continue` can cause confusion inside a `for loop`.

Reply:
Looks all good now. Thanks.